### PR TITLE
feat: add information to cascading pull request

### PR DESCRIPTION
**Pull request by kpanot**

### CODE REVIEW

File: apps/github-cascading-app/src/cascading/cascading.ts [hunk 1]
```diff
@@ -251,9 +239,41 @@ export abstract class Cascading {
...
+    const message = await this.generatePullRequestBody({
+      ...context,
+      bypassReviewers: false
+    }, pullRequest);
+
+    return this.updatePullRequestMessage(pullRequest.id, message);
+  }
+
+  /**
+   * Update the pull request message if the pull request can be merged
```
Comment (cpaulve-1A) [inline apps/github-cascading-app/src/cascading/cascading.ts:258]: Not sure about it does what your description says.

Comment (kpanot) [reply apps/github-cascading-app/src/cascading/cascading.ts:258]: missing a `not` indeed

File: apps/github-cascading-app/src/cascading/templates/pull-request-content.ejs [hunk 2]
```diff
@@ -7,17 +7,43 @@ The configuration requests the cascading to bypass reviewer in case of CI succes
...
+  gh pr checkout <%= id %>
+  git pull --ff origin <%= targetBranch %>
+  ```
+</details>
+
+<details>
+  <summary>Using `git` only</summary>
+
+  ```shell
+  git checkout origin/<%= originBranchName %>
```
Comment (matthieu-crouzet) [inline apps/github-cascading-app/src/cascading/templates/pull-request-content.ejs:40]:
```suggestion
  git fetch origin
  git checkout origin/<%= originBranchName %>
```

**Review by matthieu-crouzet — APPROVED** [review]

File: apps/github-cascading-app/src/cascading/cascading.spec.ts [hunk 3]
```diff
@@ -23,20 +23,23 @@ const mockBasicTemplate = `
 Super footer
 `;
 
+// const fakePullRequestInforObject: CascadingPullRequestInfo = { body: '', id: 0, isOpen: true, mergeable: true, originBranchName: '' };
```
Comment (cpaulve-1A) [inline apps/github-cascading-app/src/cascading/cascading.spec.ts:26]: Do you want to keep this ?

File: .eslintignore [hunk 4]
```diff
@@ -4,7 +4,9 @@ test/
 tmp/
 generated-doc/
 /{packages,apps}/@*/*/dist*/**
+/{packages,apps}/@*/*/dist*/**
```
Comment (fpaul-1A) [inline .eslintignore:7]: duplicate ?

**Review by cpaulve-1A — DISMISSED** [review]

**Review by fpaul-1A — APPROVED** [review]

**Review by fpaul-1A — APPROVED** [review]

**Review by matthieu-crouzet — APPROVED** [review]

**Pull request merged**